spec/nine_one_one_spec.rb

```diff
@@ -171,33 +98,109 @@
         end.to raise_error(NineOneOne::ConfigurationError)
       end
     end
+
+    context 'multi config' do
+      it 'allows to have separate configs for different purposes' do
+        NineOneOne.configure(:urgent) do |config|
+          config.slack_enabled = false
+        end
+
+        NineOneOne.configure(:non_urgent) do |config|
+          config.slack_webhook_url = 'webhook'
+          config.slack_channel = 'channel'
+          config.slack_enabled = true
+        end
+
+        expect(NineOneOne.configs[:default].slack_enabled).to eq(false)
+        expect(NineOneOne.configs[:default].slack_enabled).to eq(false)
```

Comment:
duplication or you wanted to check 'channel' here

Reply:
I wanted to check `expect(NineOneOne.configs[:urgent].slack_enabled).to eq(false)`. Nice catch.